README.md

```diff
@@ -28,16 +28,19 @@ The steps to validate the request for a token are:
 3. validate the the signature is from an EC2 instance
 4. Validate the document has all of the necessary information to proceed
 5. Fetch metadata from S3 that matches data from the json document
-6. acquire a token for vault that grants access to the specific secrets
+6. acquire a token for Vault that grants access to the specific secrets
 7. return the token to the instance requesting a token
 
 ## Usage
 
-First insure that ruby 2.2.4 and node 4.4.0 are installed on your vault servers.
+See the [getting started guide][gsg] for help installing, configuring, and
+using Warden.
+
+First ensure that ruby 2.2.4 and node 4.4.x are installed on your Vault servers.
 Then ensure that the settings in /config/defaults.json are correct for your needs.
 
-to launch the warden server, be in the root directory and call ```node bin/server.js```
-This will launch both warden and the Sinatra app to verify signatures of requests.
+to launch the Warden server, be in the root directory and call ```node bin/server.js```
```

Comment:
Instead of "root directory", can you be more specific?  Where does the deb install to?  Can you give the fully qualified path so losers like me can copy/paste?